drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -335,21 +336,21 @@ function DragAndDropTemplates(configuration) {
         );
     };
 
-    var sidebarButtonTemplate = function(buttonClass, iconClass, buttonText, disabled, spinner) {
-        if (spinner) {
+    var sidebarButtonTemplate = function(buttonClass, iconClass, buttonText, options) {
```

Comment:
I like the small refactor here - changing it to take an options object rather than continually add more and more parameters.